reports/005-artpark-devops-intern/report.md

```
@@ -0,0 +1,36 @@
+# Evaluation: ARTPARK @ IISc - GenAI DevOps Intern
+
+**URL:** https://artpark.in/careers
+**Legitimacy:** High Confidence (Affiliated with IISc, legitimate deep-tech incubator)
+**Score:** 4.7/5
+
+## Block A: Match con CV
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)